services/event-application-service/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rlevkovych/lohfindercpp:latest
+
+COPY . /app
+# Building an executable.
+RUN mkdir -p /app/build
+RUN cmake -S /app -B /app/build -G Ninja -D CMAKE_TOOLCHAIN_FILE=/deps/vcpkg/scripts/buildsystems/vcpkg.cmake -D CMAKE_BUILD_TYPE="Release"
+RUN cmake --build /app/build
```

Comment:
are you ware that it would create 3 images? can we combine cmakes with &?

Reply:
Nope, it will create 3 layers for the image. It is completely ok.